OOP_HW1/src/test/java/ut_ticketsystem.java

```diff
@@ -0,0 +1,227 @@
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowable;
+
+import java.util.Vector;
+
+public class ut_ticketsystem {
+    @Test
+    public void test_check_ticket_price_should_have_correct_price() {
+
+        Order ticket = new Order(TicketType.ADULT, 1);
+        int price = TicketSystem.CheckTicketPrice(ticket);
+
+        assertThat(price).matches(q -> q == 350);
+    }
+
+    @Test
+    public void test_check_ticket_price_should_throw_exception() {
+
+        Order ticket = new Order();
+        ticket.setTicketType(TicketType.ADULT);
+        ticket.setQuantity(-1);
+
+        Throwable thrown = catchThrowable(() -> {
+            TicketSystem.CheckTicketPrice(ticket);
+        });
+
+        assertThat(thrown)
+                .isInstanceOf(IllegalArgumentException.class)
+                .hasMessageContaining("There is at least one ticket.");
```

Comment:
In assertj, there's have more simplify statement to assert a throwable.

Here's the suggestion for you:
```suggestion
        assertThatThrownBy(() -> TicketSystem.CheckTicketPrice(ticket)).hasMessage("There is at least one ticket.");
```